CONTRIBUTING.es.md

```diff
@@ -97,13 +97,13 @@ Cuando escribas tu publicación, necesitarás incluir metadatos en el *frontmatt
 <details>
   <summary><strong>Propiedades opcionales</strong></summary>
 
-  Existen algunas propiedades extra que *podrías* incluir en el frontmatter de tu publicación, pero que no son necesarias:
+  Existen algunas propiedades de sobra que son possibles incluir *como opcion* en el frontmatter de tu publicación, pero que no son necesarias:
```

Comment:
I think `podrías` is also a more accurate translation here. I also changed `de sobra` to `adicionales`.

```suggestion
  Existen algunas propiedades adicionales que *podrías* incluir en el frontmatter de tu publicación, pero que no son necesarias:
```